### PR TITLE
Refactor: Costs component

### PR DESCRIPTION
**Pull request by linds0202**
Create IconWithTextXL component and style with Tailwind CSS 

- [x] Create IconWithTextXL component 
- [x] Style with Tailwind CSS 
- [x] Refactor IconWithText component positionOfVisual w/helper functions
- [x] Write component tests

Working on component tests for description as react node  and positionOfVisual

### CODE REVIEW

**Comment by jtfairbank** [comment]
Hey @linds0202 is this one ready for review?  it seems like you've made quite a lot of progress!  if so, can you mark it as ready on your end ("remove draft status" or something)?

**Comment by linds0202** [comment]
I refactored the iconWithText component and wrote tests, I think this is ready for review

File: src/components/icon/IconWithText.test.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    expect(label).toBeTruthy()
+    expect(image).toBeTruthy()
+  })
+
+  it('Align content to the top or left', () => {
+    let contentAlignmentTop = getFlexAlignment('top')
+    let contentAlignmentLeft = getFlexAlignment('left')
+
+    expect(contentAlignmentTop).toBe(
+      'flex-wrap flex-col content-around w-1/2 h-40',
```
Comment (jtfairbank) [inline src/components/icon/IconWithText.test.tsx:26]: Using [conventional comments](https://conventionalcomments.org/) style feedback:

- polish(non-blocking): Doing an exact comparison here seems brittle.  If styles unrelated to the flex alignment change, then this test will break.  In the future I'd recommend checking that the string contains the key differences between options (ie. `top => flex-col-content-around` & `left => justify-around`)

**Review by jtfairbank — APPROVED** [review]
Using [conventional comments](https://conventionalcomments.org/) style feedback:

 - praise: looks great @linds0202!  Going to merge this now, do take a peek at the `polish` comment I left on one of your tests. :😸

**Pull request merged**